actioncable.go

```diff
@@ -28,6 +28,7 @@ type Client struct {
 
 	// TODO: what actually needs a mutex??
 	mu            sync.Mutex
+	subscData     map[string]json.RawMessage
```

Comment:
I would maybe just call this `subData`, the "sc" at the end makes me think this is a channel at glance. Small stylistic preference I guess :)